### DISCUSSION TITLE
Ideas for Denver Devs site features

### DISCUSSION THREAD

**Discussion by danhannigan** [Denver-Devs/denverdevs.org#124] [Ideas]
Just dropping some ideas out here that I've had over time for some possible cool features: 

### General Site Related:
- [ ] RSS feed for updates 
- [ ] Move Discord Updates to the website and post to Discord when they're published
- [x] Make the updates markdown friendly!
- [ ] Automatically build and deploy each night (used to do this on the old site, easy to set up)
- [ ] Pull in RSS feeds from Denver Devs authors
~~Display latest jobs, meetups, and blogs on the homepage of the website + their own page~~
- [ ] Create a public change log and start versioning out the site
- [x] Fix / Improve bad SEO (I don't think I have headers or descriptions on like... any page) 
- [ ] Make a nice 404 page.

### Meetup / Event Related:
**WONT DO, MEETUP API CHANGED. ITS $30+ A MONTH NOW**
~~Pull in Meetup.com meetups (did this on the old site, too!)~~
~~Use Supabase to make a database of Meetups + Events that aren't on Meetup or anywhere else (with a dashboard for users to CRUD their meetups)~~
~~Let users mark if they're interested in going to the meetup (see if we can pull this from Meetup.com and also allow it on our own posts) and show interested users on the post~~

### Job Board Related:
- [ ] RSS feed(s) for Job Posts
- [x] Use static site gen on job listing page since jobs have to be manually approved anyway, can kick off a build after
- [x] Build a page for each job so it has it's own unique URL + SEO 
- [ ] Add in an "Edit" button some time down the road
- [x] Allow for a brief description of the job (if we build individual job pages) 
- [ ] Refine the formatting for the Job Cards on mobile + desktop
- [x] Figure out how to make the filters look better (maybe horizontal filter bar up top?)
- [x] Allow filtering by Location
- [ ] Let users "save" or "favorite" jobs (would require them to sign in)
- [ ] Display how many users have saved/favorited a job

**Comment by tbeseda** [comment]
I like more connection between the site and the Discord server 👍 Things like events (AMAs etc) and meetups could sync. Or a way to show and tell from #today-i maybe? +1 for DD author feed. Even if it's just a link log.
I rarely have reason to visit the site unless it's to check out your handy work. Happy to help on integration type things.
